cmd/tools/commentMonitor/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/v26/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+const prombenchURL = "http://prombench.prometheus.io"
+
+func newClient(token string) *github.Client {
+	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
+	tc := oauth2.NewClient(context.Background(), ts)
+	clt := github.NewClient(tc)
+	return clt
+}
+func memberValidation(authorAssociation string) error {
+	if (authorAssociation != "COLLABORATOR") && (authorAssociation != "MEMBER") {
+		return fmt.Errorf("not a member or collaborator")
+	}
+	return nil
+}
+
+func regexValidation(regex string, comment string) ([]string, error) {
+	argRe := regexp.MustCompile(regex)
+	if argRe.MatchString(comment) {
+		arglist := argRe.FindStringSubmatch(comment)
+		return arglist, nil
+	}
+	return []string{}, fmt.Errorf("invalid command")
+}
+
+func writeArgs(args []string, output string) {
+	for i, arg := range args[1:] {
+		data := []byte(arg)
+		filename := fmt.Sprintf("ARG_%v", i)
+		err := ioutil.WriteFile(filepath.Join(output, filename), data, 0644)
+		if err != nil {
+			log.Fatalln(err)
+		}
+		log.Printf(filepath.Join(output, filename))
+	}
+}
+
+func postComment(client *github.Client, owner string, repo string, prnumber int, comment string) error {
+	issueComment := &github.IssueComment{Body: github.String(comment)}
+	_, _, err := client.Issues.CreateComment(context.Background(), owner, repo, prnumber, issueComment)
+	return err
+}
+
+func main() {
+	app := kingpin.New(filepath.Base(os.Args[0]), "commentMonitor github comment extract")
+	app.HelpFlag.Short('h')
+	input := app.Flag("input", "path to event.json").Short('i').Default("/github/workflow/event.json").String()
+	output := app.Flag("output", "path to write args to").Short('o').Default("/github/home").String()
+	regex := app.Arg("regex", "Regex pattern to match").Required().String()
+
+	kingpin.MustParse(app.Parse(os.Args[1:]))
```

Comment:
I think the prombench cli prints some examples when you type a wrong command or with the -h flag so meed to add the same here

Reply:
Prombench cli prints examples for the subcommands and not for the flags. Here, we only have flags. Do we still need to print examples?

Reply:
Yes i think one example is good to have

Reply:
don't you need something like:

https://github.com/prometheus/prombench/blob/a926df5a16e0eb992bf8f71a74670293e7aa72fb/cmd/prombench/prombench.go#L60-L64